libs/frc20_abi/Forc.toml

```diff
@@ -0,0 +1,5 @@
+[project]
+authors = ["Fuel Labs <contact@fuel.sh>"]
+entry = "lib.sw"
+license = "Apache-2.0"
+name = "frc20_abi"
```

Comment:
Interesting name, `frc`. Also, why 20? Are we following Ethereum for a specific reason or can we have some other convention?

Reply:
yeah, reserving all the common numbers for parallels to Ethereum makes sense to me